llama_stack/providers/inline/agents/meta_reference/agent_instance.py

```diff
@@ -459,7 +455,15 @@ async def _run(
                 contexts.append(raw_document_text)
 
             attached_context = "\n".join(contexts)
-            input_messages[-1].context = attached_context
+            if isinstance(input_messages[-1].content, str):
+                input_messages[-1].content += attached_context
```

Comment:
dont you have to add the attached_context at the beginning of the content? not append it? Also a little lost about what the shape of content is in general - will be good to provide some examples.

Reply:
This follows what is done in llama-models, where the "context" is appended after "content". 

https://github.com/meta-llama/llama-models/blob/ee8ede3deb42dbfc8e151fd303c1cd9acdfe1869/models/llama3/api/chat_format.py#L117-L141

The raw message sent to provider has context after content. 
<img width="825" alt="image" src="https://github.com/user-attachments/assets/8962b422-3235-46f0-bd2a-a53c1fb6b3bb" />

There could be further improvements and additional configuration to do prompt engineering for appending context, but that's outside scope of this PR.